packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -190,8 +193,40 @@ export default defineComponent({
   name: 'FileDetails',
   setup() {
     const store = useStore()
+    const { $gettext } = useTranslations()
+
+    const copiedDirect = ref(false)
+    const copiedEos = ref(false)
+    const { copy, copied, isSupported } = useClipboard({ legacy: true, copiedDuring: 550 })
+
+    const copyEosPathToClipboard = () => {
+      copy(inject<ComputedRef<Resource>>('displayedItem').value.path)
+      copiedEos.value = copied.value
+      store.dispatch('showMessage', {
+        title: $gettext('EOS path copied'),
+        desc: $gettext('The EOS path has been copied to your clipboard.')
+      })
+    }
```

Comment:
I know you just copied existing code, but do we really want to expose the `EOS` string/name to a user? 

Should we revisit this, while you're at it?

Reply:
> Perfect, obviously 💫

I'm not quite sure if I get the `isSupported` right though - reading https://github.com/vueuse/vueuse/blob/70f55d4e68fd8f100fe844dd883c89b6d6594d03/packages/core/useClipboard/index.ts#L67 it seems like I could omit it since it is basically supported through the legacy option anyways?

Now it's "only" fixing unit tests, no opinion about the EOS translation thought...it's a CERN-internal thing anyways and practically no-one will be able to "casually spin up Reva/oCIS/oc-web on an EOS and then be confused about those strings" ;P

Reply:
And from what I remember they literally send each other EOS paths when referencing resources